gulpfile.coffee

```diff
@@ -122,30 +119,29 @@ gulp.task('install.rdf', ->
     .pipe(gulp.dest(DEST))
 )
 
-gulp.task('templates', [
+gulp.task('templates', gulp.series(
   'bootstrap-frame.js'
   'chrome.manifest'
   'install.rdf'
-])
+))
 
-gulp.task('build', (callback) ->
-  runSequence(
-    'clean',
-    ['copy', 'node_modules', 'coffee', 'templates'],
-    callback
-  )
-)
+gulp.task('build', gulp.series(
+  'clean',
+  gulp.parallel('copy', 'node_modules', 'coffee', 'templates')
+))
 
-gulp.task('xpi', ['build'], ->
+gulp.task('xpi', gulp.series('build', ->
   gulp.src("#{DEST}/**/*")
     .pipe(zip(XPI, {compress: false}))
     .pipe(gulp.dest(DEST))
-)
+))
 
-gulp.task('push', ['xpi'], ->
+gulp.task('push', gulp.series('xpi', ->
   body = fs.readFileSync(join(DEST, XPI))
   request.post({url: 'http://localhost:8888', body})
-)
+))
+
+gulp.task('default', gulp.series(['push']))
```

Comment:
Is `gulp.series` supposed to be given an array here? The other `gulp.series` are not given arrays.

Reply:
My mistake. Fixed.